src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -183,6 +182,25 @@ public void localizationParameterMustIncludeAString() throws IOException {
         }
     }
 
+    @Test
+    public void localizationTestForInvalidStrings() {
+        for (String bundle : Arrays.asList("JabRef", "Menu")) {
+            for (String lang : Languages.LANGUAGES.values()) {
+                String propertyFilePath = String.format("/l10n/%s_%s.properties", bundle, lang);
+
+                // read in
+                Properties textKeys = LocalizationParser
+                        .getProperties(propertyFilePath);
+
+                //parse object "textKeys" to find any spaces
+                for (Map.Entry<Object, Object> entry : textKeys.entrySet()) {
+                    assertFalse("Found an invalid character in the " + lang + " localization of " + bundle + " : " + entry.getValue().toString() + " At key : " + entry.getKey().toString() + " contains a space!", entry.getValue().toString().contains(" "));
```

Comment:
Just a minor thing: if you concatenate Strings and objects (no matter which type) with` "some text " + object    `
You don't need to do an explicit call to "toString()" The method is automatically invoked. But beware, in the case where you want to call a methond on a string like the contains, you have to use the toString

Reply:
Thank you @Siedlerchr for this explanation.